packages/@webex/internal-plugin-scheduler/package.json

```diff
@@ -38,6 +38,7 @@
     "@webex/webex-core": "workspace:^",
     "ampersand-collection": "^2.0.2",
     "ampersand-state": "^5.0.3",
+    "btoa": "^1.2.1",
```

Comment:
Can this be dependency be removed? Functions within the [base64.js](https://github.com/webex/webex-js-sdk/blob/master/packages/%40webex/common/src/base64.js) utilities file should suffice, I believe.

Reply:
OK, I will remove it.